### ISSUE TITLE
This project still active?

### ISSUE THREAD

**Issue by andrekutianski** [srtucker22/quasar#17]
The quasar project still active?

**Comment by srtucker22** [comment]
Yes -- sorta. There's a [branch](https://github.com/srtucker22/quasar/tree/redux) I'll be merging to master in the near future which switches this app to meteor + redux (and a rename for the app). I haven't had a chance to test on android and I'm without a phone to test atm so I'm a bit hesitant to make the push right away.

I'm also in the process of making a similar app with [apollo](http://www.apollodata.com/) and React Native as I'm finding Cordova to be a bit slow, and also unreliable with WebRTC on iOS, and this new stack is *super slick*

Feel free to let me know your interest and I might be able to help.

**Comment by untotren** [comment]
Please dont stop this is aweosme!

**Comment by srtucker22** [comment]
i promise it'll get a proper revamp soon!

my free time is wrapped up in contributing to [apollo](http://www.apollodata.com/) and writing the  [chatty](https://medium.com/@simontucker/building-chatty-a-whatsapp-clone-with-react-native-and-apollo-part-1-setup-68a02f7e11) tutorial atm, so bear with me....

**Comment by srtucker22** [comment]
The latest major overhaul is released! Hoping to revamp and reimplement Android Cordova support in the near future. Please feel free to play with the latest and contribute if you're up for it. Will try and keep a steady cadence up updates moving forward.
